CTFd/templates/admin/teams.html

```diff
@@ -123,10 +123,10 @@ <h2 class="text-center">Edit User</h2>
             {% for team in teams %}
             <tr name="{{ team.id }}">
                 <td class="team-id">{{ team.id }}</td>
-                <td class="team-name"><a href="/admin/team/{{ team.id }}">{{ team.name | truncate(32) }}</a>
+                <td class="team-name"><a href="{{ request.script_root }}/admin/team/{{ team.id }}">{{ team.name | truncate(32) }}</a>
                 </td>
                 <td class="team-email">{{ team.email | truncate(32) }}</td>
-                <td class="team-website">{% if team.website and team.website.startswith('http') %}<a href="{{ team.website }}">{{ team.website | truncate(32) }}</a>{% endif %}
+                <td class="team-website">{% if team.website and team.website.startswith('http') %}<a href="{{ request.script_root }}{{ team.website }}">{{ team.website | truncate(32) }}</a>{% endif %}
```

Comment:
This should not have a `{{ request.script_root }}` as it's a user provided website.